app/views/components/goodsLocationExpander.scala.html

```diff
@@ -0,0 +1,135 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
+
+@this(govukDetails : GovukDetails, link: link, paragraph: paragraphBody, appConfig: AppConfig)
+
+@()(implicit messages: Messages)
+
+@expanderContent = {
+    @paragraph(messages("declaration.goodsLocation.expander.intro"))
+
+    <p><b>@messages("declaration.goodsLocation.expander.paragraph1.title")</b></p>
+    <div id="code-hint" class="govuk-hint">
+    @Html(messages("declaration.goodsLocation.expander.paragraph1.text",
+        link(
+            text = messages("declaration.goodsLocation.expander.paragraph1.link1.text"),
+            call = Call("GET", appConfig.companyInformationRegister),
+            target = "_new"
```

Comment:
`_new` ? Do you have an `iframe` in this page? :thinking:

Reply:
iframe?? Not sure about that but back in the day "_new" was considered ok, looks like things have changed though as I've now read it must be "_blank" according to HTML5 spec. Updated accordingly :slightly_smiling_face: